memfs/memory_option.go

```diff
@@ -0,0 +1,6 @@
+package memfs
+
+type Option func(*options)
+
+type options struct {
+}
```

Comment:
We can remove this altogether
```suggestion
```

Reply:
I left this on purpose, the idea is to add additional options on a follow-up PR. Please note that the pattern aligns with both osfs implementations.